cves/CVE-2010-1623.yml

```diff
@@ -26,7 +26,6 @@ announced_instructions: |
   (https://chromereleases.googleblog.com/).
   Please enter your date in YYYY-MM-DD format.
 announced: 2010-10-04T21:00Z
-
```

Comment:
Why was this modified?

Reply:
I think it was my editor added stuff. It was un intentional